## REQUEST

# The target goal for the drone
float64 x
float64 y
float64 z
float64 yaw
